console/my-account/concepts.mdx

```diff
@@ -53,14 +45,6 @@ The Scaleway Console allows you to view and manage your Scaleway products, billi
 
 <Concept>
 
-## Credentials
-
-Credentials are unique to each [project](#project), and consist of unique and secure identifiers to authenticate your connections and commands to your Scaleway products. Credentials include [SSH Keys](#ssh-key) and [API Keys](#api-key).
-
-</Concept>
-
-<Concept>
-
 ## Default project
 
 Each [Organization](#organization) must have at least one associated [Project](#project), and at the time of account creation this project is called **default**. The default project takes on the Organization ID. Therefore, the default status cannot be transferred to other projects, nor can the default project's name be altered.
```

Comment:
The link to #project no longer works. Either:

- remove this whole "Default project" concept (it already exists in My Project)
- leave the Project concept on this page
- add the Project concept to My Project and link to it there

Reply:
DELETE CONCEPT